integration/utils/batchLoadGenerator.go

```diff
@@ -354,7 +354,7 @@ func (lg *BatchLoadGenerator) rotateTokens() error {
 		j := (i + 1) % len(lg.accounts)
 		transferScript, err := TokenTransferScript(
 			lg.fungibleTokenAddress,
-			lg.accounts[i].address,
+			lg.flowTokenAddress,
```

Comment:
we never pass the address of the source to `TokenTransferScript` as it always uses the payer as the source of the funds. but we do need the address of the account in which FlowToken is deployed.